packages/manager/modules/pci/src/projects/project/training/jobs/info/info.html

```diff
@@ -5,14 +5,14 @@
     <oui-header-tabs>
         <oui-header-tabs-item
             data-active="$ctrl.currentActiveLink() === $ctrl.jobInfo"
-            data-href="{{:: $ctrl.jobInfo }}"
+            data-href="{{   $ctrl.jobInfo }}"
```

Comment:
Some one time expressions has been removed but leads with some extra space.
Could you please remove them?.

Example:
```suggestion
            data-href="{{ $ctrl.jobInfo }}"
```

The entire file is affected.
Thanks!